src/core/forms/forms.py

```diff
@@ -480,7 +480,7 @@ class Meta:
         fields = [
             'name', 'plural', 'number_of_reviewers',
             'is_filterable', 'sequence', 'section_editors',
-            'editors', 'public_submissions', 'indexing',
+            'editors', 'jats_article_type', 'public_submissions', 'indexing',
```

Comment:
should this be `jats_article_type_override` now?

Reply:
No: this is for the section, which is not an override